Makefile

```diff
@@ -1,16 +1,13 @@
-NS ?= ghcr.io/playfab
+
+NS ?= ghcr.io/playfab/
 
 export IMAGE_NAME_OPERATOR=thundernetes-operator
 export IMAGE_NAME_NODE_AGENT=thundernetes-nodeagent
 export IMAGE_NAME_INIT_CONTAINER=thundernetes-initcontainer
-export IMAGE_NAME_NETCORE_SAMPLE=thundernetes-netcore-sample
-export IMAGE_NAME_OPENARENA_SAMPLE=thundernetes-openarena-sample
+export IMAGE_NAME_NETCORE_SAMPLE=thundernetes-netcore
+export IMAGE_NAME_OPENARENA_SAMPLE=thundernetes-openarena
 
-export OPERATOR_TAG?=$(shell git rev-list HEAD --max-count=1 --abbrev-commit)
-export NODE_AGENT_TAG?=$(shell git rev-list HEAD --max-count=1 --abbrev-commit)
-export INIT_CONTAINER_TAG?=$(shell git rev-list HEAD --max-count=1 --abbrev-commit)
-export NETCORE_SAMPLE_TAG?=$(shell git rev-list HEAD --max-count=1 --abbrev-commit)
-export OPENARENA_SAMPLE_TAG?=$(shell git rev-list HEAD --max-count=1 --abbrev-commit)
+export IMAGE_TAG?=$(shell git rev-list HEAD --max-count=1 --abbrev-commit)

```

Comment:
any chance we can want different image tags (versions) for each of the projects?

Reply:
I can't see why we would, since we are building them from the same repo, using this Makefile :)

Reply:
In case we have any small bug fix, for example? Which does prevent us from updating all the images with the new tag though

Reply:
I'm not sure I'm completely following, but even if say we were fixing something in just the init container; all the containers get rebuilt by the makefile anyways, so there's not a need to have different tag variables.

As an aside, I'm a fan of having "matching tags" for the containers so that a consumer knows they all came from the same revision/build/version.

Reply:
Sorry, I forgot a “not” in my response, hope it’s more clear now :) I’m good, thanks for the clarifications!